### PR TITLE
Version/1.0 master merge

### PR DESCRIPTION
**Pull request by rrennick**
This week's merge of `version/1.0` into `master`

- Review code changes
- Smoke test screens

### CODE REVIEW

File: client/analytics/report/index.js [hunk 1]
```diff
@@ -94,6 +94,19 @@ export const getReports = () => {
...
+/**
+ * The Customers Report will not have the `report` param supplied by the router/
+ * because it no longer exists under the path `/analytics/:report`. Use `props.path`/
+ * instead to determine if the Customers Report is being rendered.
+ *
+ * @param {Object} params - url parameters
+ * @param {string} path - url path
+ * @return {string} - report parameter
+ */
+const getReportParam = ( params, path ) => {
```
Comment (psealock) [inline client/analytics/report/index.js]:
```suggestion
const getReportParam = ( { params, path } ) => {
```
Reports are breaking due to this 😄

File: client/analytics/report/index.js [hunk 2]
```diff
@@ -94,6 +94,19 @@ export const getReports = () => {
 	return applyFilters( REPORTS_FILTER, reports );
 };
 
+/**
+ * The Customers Report will not have the `report` param supplied by the router/
+ * because it no longer exists under the path `/analytics/:report`. Use `props.path`/
+ * instead to determine if the Customers Report is being rendered.
+ *
+ * @param {Object} params - url parameters
```
Comment (psealock) [inline client/analytics/report/index.js]:
```suggestion
 * @param {Object} props - component props
```

File: client/analytics/report/index.js [hunk 3]
```diff
@@ -94,6 +94,19 @@ export const getReports = () => {
...
+ * @param {string} path - url path
```
Comment (psealock) [inline client/analytics/report/index.js]:
```suggestion
```

**Comment by rrennick** [comment]
@psealock Thanks for the review. I found one block of code that I had mis-merged. Can you give this another check?

**Comment by timmyc** [comment]
@rrennick thanks so much for doing this. Curious about how long it took you? Trying to get a feel for the level of effort for this approach vs cherrypicking...

**Comment by rrennick** [comment]
> Curious about how long it took you?

The next time will probably take 3-4 hours. This time I got stuck on the customer search not working in my install where I spent quite a bit of time before finding it was my install.

**Pull request merged**